builder/dockerfile/dispatchers_test.go

```diff
@@ -215,49 +215,49 @@ func TestFromScratch(t *testing.T) {
 	err := from(b, []string{"scratch"}, nil, "")
 
 	if runtime.GOOS == "windows" {
-		assert.Error(t, err, "Windows does not support FROM scratch")
+		assert.EqualError(t, err, "Windows does not support FROM scratch")
 		return
 	}
 
-	assert.NilError(t, err)
-	assert.Equal(t, b.image, "")
-	assert.Equal(t, b.noBaseImage, true)
+	assert.NoError(t, err)
+	assert.Equal(t, "", b.image)
+	assert.Equal(t, true, b.noBaseImage)
 }
 
 func TestFromWithArg(t *testing.T) {
 	tag, expected := ":sometag", "expectedthisid"
 
 	getImage := func(name string) (builder.Image, error) {
-		assert.Equal(t, name, "alpine"+tag)
+		assert.Equal(t, "alpine"+tag, name)
 		return &mockImage{id: "expectedthisid"}, nil
 	}
 	b := newBuilderWithMockBackend()
 	b.docker.(*MockBackend).getImageOnBuildFunc = getImage
 
-	assert.NilError(t, arg(b, []string{"THETAG=" + tag}, nil, ""))
+	assert.NoError(t, arg(b, []string{"THETAG=" + tag}, nil, ""))
 	err := from(b, []string{"alpine${THETAG}"}, nil, "")
 
-	assert.NilError(t, err)
-	assert.Equal(t, b.image, expected)
-	assert.Equal(t, b.from.ImageID(), expected)
-	assert.Equal(t, len(b.buildArgs.GetAllAllowed()), 0)
-	assert.Equal(t, len(b.buildArgs.GetAllMeta()), 1)
+	assert.NoError(t, err)
+	assert.Equal(t, expected, b.image)
+	assert.Equal(t, expected, b.from.ImageID())
+	assert.Len(t, b.buildArgs.GetAllAllowed(), 0)
+	assert.Len(t, b.buildArgs.GetAllMeta(), 1)
```

Comment:
I think this is a good example of an inconsistency in the design of testify. 
`assert.Len()`, `assert.Contains()`, and `assert.EqualError()` have actual as the first arg, and expected as the second.

The args for `assert.Equal(t, expected, actual)`  are in the opposite order. They should have been `assert.Equal(t, actual, expected)` to be consistent with the other assertions.

However I don't think that's enough of a reason to keep our own implementation.